docs: fix typos and grammar in CONTRIBUTING.md (#10181)
dapr/dapr: CONTRIBUTING.md

Thank you for your interest in Dapr!

This project welcomes contributions and suggestions. Most contributions require you to sign off on your commits via 
the Developer Certificate of Origin (DCO). When you submit a pull request, a DCO-bot will automatically determine 

> whether you need to provide sign-off for your commit. Please follow the instructions provided by DCO-bot, as pull 

Reviewer: ```suggestion
whether you need to provide sign off for your commit. Please follow the instructions provided by DCO-bot, as pull 
```

> whether you need to provide sign off for your commit. Please follow the instructions provided by DCO-bot, as pull 

Reviewer 2: ```suggestion
whether you need to sign off on your commits. Please follow the instructions provided by DCO-bot, as pull 
```

> requests cannot be merged until the author(s) have provided sign-off to fulfill the DCO requirement.

Reviewer: ```suggestion
requests cannot be merged until the author(s) have provided sign off to fulfill the DCO requirement.
```

> requests cannot be merged until the author(s) have provided sign off to fulfill the DCO requirement.

Reviewer 2: ```suggestion
requests cannot be merged until the author(s) have fulfilled the DCO requirement.
```
